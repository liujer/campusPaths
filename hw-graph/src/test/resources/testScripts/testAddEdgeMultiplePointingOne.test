# Tests adding of edges by having multiple
# nodes point to one node
CreateGraph E
AddNode E one
AddNode E two
AddNode E three
AddEdge E one two e12
AddEdge E three two e32
ListNodes E
ListChildren E one
ListChildren E two
ListChildren E three