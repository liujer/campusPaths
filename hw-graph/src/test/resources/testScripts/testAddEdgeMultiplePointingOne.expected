# Tests adding of edges by having multiple
# nodes point to one node
Created graph E
Added node one to E
Added node two to E
Added node three to E
Added edge e12 to E
Added edge e32 to E
ListNodes of E output: one three two
ListChildren of one in E output: two
ListChildren of two in E output:
ListChildren of three in E output: two